clang/lib/AST/ByteCode/InterpBuiltin.cpp

```diff
@@ -2937,6 +2937,50 @@ static bool interp__builtin_elementwise_triop(
   return true;
 }
 
+static bool interp__builtin_x86_insert_subvector(InterpState &S, CodePtr OpPC,
+                                                 const CallExpr *Call,
+                                                 unsigned ID) {
+  assert(Call->getNumArgs() == 3);
+
+  PrimType ImmPT = *S.getContext().classify(Call->getArg(2));
+  APSInt ImmAPS = popToAPSInt(S.Stk, ImmPT);
```

Comment:
```suggestion
  PrimType ImmPT = *S.getContext().classify(Call->getArg(2));
  APSInt ImmAPS = popToAPSInt(S, Call->getArg(2));
```
You need to rebase for that though.